lib/src/downloader.dart

```diff
@@ -58,14 +73,26 @@ Future<RevisionInfo> downloadChrome({int? revision, String? cachePath}) async {
       revision: revision);
 }
 
-Future _downloadFile(String url, String output) async {
-  var client = http.Client();
-  var response = await client.send(http.Request('get', Uri.parse(url)));
-  var ouputFile = File(output);
-  await response.stream.pipe(ouputFile.openWrite());
-  client.close();
+Future _downloadFile(
+  String url,
+  String output,
+  Function(int, int?)? onRecieveProgress,
+) async {
+  final client = http.Client();
+  final response = await client.send(http.Request('get', Uri.parse(url)));
+  final totalBytes = response.contentLength;
```

Comment:
I wasn't sure how to handle the case if the the size of the request is not known in advance. 
`totalBytes' will be null in that case.

Reply:
I think defaulting to zero is good enough. I pushed a commit on your branch with that.

Reply:
Okay, sounds good!